queue.tf

```diff
@@ -20,12 +20,14 @@ module "envelopes-queue" {
 }
 
 module "notifications-queue" {
-  source              = "git@github.com:hmcts/terraform-module-servicebus-queue?ref=master"
-  name                = "notifications"
-  namespace_name      = "${module.queue-namespace.name}"
-  resource_group_name = "${azurerm_resource_group.rg.name}"
-  lock_duration       = "PT5M"
-  max_delivery_count  = "${var.notification_queue_max_delivery_count}"
+  source                                  = "git@github.com:hmcts/terraform-module-servicebus-queue?ref=master"
+  name                                    = "notifications"
+  namespace_name                          = "${module.queue-namespace.name}"
+  resource_group_name                     = "${azurerm_resource_group.rg.name}"
+  lock_duration                           = "PT5M"
+  max_delivery_count                      = "${var.notification_queue_max_delivery_count}"
```

Comment:
imho git history matters, let's revert those as it's not related

Reply:
reverted formatting changes @rkondratowicz